typescript/sdk/src/token/schemas.ts

```diff
@@ -63,12 +48,36 @@ export const TokenConfigSchema = z.discriminatedUnion('type', [
   SyntheticConfigSchema,
 ]);
 
-export const TokenRouterConfigSchema = z.intersection(
-  TokenConfigSchema,
-  routerConfigSchema,
+export const TokenRouterConfigSchema = TokenConfigSchema.and(
+  GasRouterConfigSchema,
 );
 
-export const WarpRouteDeployConfigSchema = z.record(
-  z.string(),
-  TokenRouterConfigSchema,
-);
+export const WarpRouteDeployConfigSchema = z
+  .record(TokenRouterConfigSchema)
+  .refine((configMap) => {
+    const entries = Object.entries(configMap);
+    return (
+      entries.some(
+        ([_, config]) =>
+          CollateralConfigSchema.safeParse(config).success ||
```

Comment:
can we use `isCompliant`, or one of the helpers that you created below?

Reply:
yes :)